NBXplorer.Client/NBXplorerNetworkProvider.Liquid.cs

```diff
@@ -29,6 +29,11 @@ public override BitcoinAddress CreateAddress(DerivationStrategyBase derivationSt
 
 			public static Key GenerateBlindingKey(DerivationStrategyBase derivationStrategy, KeyPath keyPath)
```

Comment:
this should throw `InvalidOperationException` if it is unblinded.

Reply:
fixed